src/common/redis_module/ray_redis_module.cc

```diff
@@ -1429,6 +1642,18 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx,
     return REDISMODULE_ERR;
   }
 
+  if (RedisModule_CreateCommand(ctx, "ray.table_request_notifications",
```

Comment:
Side note: I probably won't be able to port all of these to credis myself...

Reply:
Yeah, I can help with that. I hope though, that this interface will actually be easier to implement in credis than trying to support everything in vanilla Ray's Redis modules.